momepy/preprocessing.py

```diff
@@ -756,6 +757,57 @@ def _polygonize_ifnone(edges, polys):
     return polys
 
 
+def _create_shape_index(gdf):
```

Comment:
There has been a development around this in the meantime leading to dropping of this specific formula for the shape index. I don't want to go into details now but the mathematical derivation of this one was flawed.

We now use this one in the paper (in progress)

```py
from esda import shape

polygons["circular_compactness"] = shape.minimum_bounding_circle_ratio(polygons)
polygons["circular_compactness_index"] = polygons["circular_compactness"] * area
```

I suggest trying the same, but the threshold values will obviously change.

For sure, we should use `esda` to measure the actual index to avoid calling pygeos directly here (we'll need to get rid of pygeos soon).

Reply:
I just reviewed this and the reality is that we only use `reock` as a selection metric for roundabouts. By using `shape.minimum_bounding_circle_ratio(polygons)` we would be simplifying the way we get those numbers (which is great) but the actual values are the same. 
Either I'm missing something or we could just make those changes and forget about the extra step to create the INDEX that in reality never gets used.